next.config.js

```diff
@@ -27,6 +27,7 @@ const nextConfig = {
       "www.frenslands.xyz",
       "raw.githubusercontent.com",
       "lh3.googleusercontent.com",
+      "starkdefi.com",
```

Comment:
```suggestion
      "www.starkdefi.com",
```
This should be the url to make it work